pandas/tests/arrays/sparse/test_arithmetics.py

```diff
@@ -29,7 +31,7 @@ class TestSparseArrayArithmetics:
     def _assert(self, a, b):
         tm.assert_numpy_array_equal(a, b)
 
-    def _check_numeric_ops(self, a, b, a_dense, b_dense, mix, op):
+    def _check_numeric_ops(self, a, b, a_dense, b_dense, mix, op, flag=False):
```

Comment:
what is flag for?

Reply:
leftover from previous attempt at a fix, will remove